virtual-setup/windows-instructions.md

```diff
@@ -1,7 +1,78 @@
-## Windows set up instructions for virtual workshops
+# Windows set up instructions for virtual workshops
 
-TOC
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
-### Zoom
+- [Windows set up instructions for virtual workshops](#windows-set-up-instructions-for-virtual-workshops)
+  - [Zoom Installation and Setup](#zoom-installation-and-setup)
+    - [New Installation](#new-installation)
+    - [Set Up Preferences](#set-up-preferences)
+  - [Slack](#slack)
 
-### Slack
\ No newline at end of file
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Zoom Installation and Setup
+
+### New Installation
+
+If you do not have Zoom installed yet, you will need to download the client.
+To do so, go to https://zoom.us/download and click the button to download the latest version of **Zoom Client for Meetings**
+
+<img src="screenshots/windows-zoom-download.png" width=500 alt="Zoom Download Page">
+
+The `Zoominstaller.exe` installer package will most likely be saved to your `Downloads` folder or you can click on it in the lower left corner.
+Click `Run` when asked what you would like to do with `Zoominstaller.exe`
+
+Windows will ask you if you would like to allow Zoom to make changes to your device. Click `Yes`.
+
+When the installation is completed, Zoom should automatically open.
+Click `Join with Video` if asked.
+
```

Comment:
Does this happen if they aren't clicking a meeting link?

Reply:
I am unsure. I wrote it down in my original notes that it happened when installation completed?